lib/Mojolicious/Guides/Testing.pod

```diff
@@ -0,0 +1,754 @@
+
+=encoding utf8
+
+=head1 NAME
+
+Mojolicious::Guides::Testing - Web Application Testing Made Easy
+
+=head1 OVERVIEW
+
+This document is an introduction to testing web applications with
+L<Test::Mojo>. L<Test::Mojo> can be thought of as a module that provides all
+of the tools and testing assertions needed to test web applications in a
+Perlish way.
+
+While L<Test::Mojo> can be used to test any web application, it has shortcuts
+designed to make testing L<Mojolicious> web applications easy and pain-free.
+
+Please refer to the L<Test::Mojo> documentation for a complete reference to
+many of the ideas and syntax introduced in this document.
+
+A test file for a simple web application might look like:
+
+  use Test::Mojo;
```

Comment:
You probably want `use Mojo::Base -strict` here or at least some strict/warnings enabled.